### ISSUE TITLE
flock(): Illegal operation argument

### ISSUE THREAD

**Issue by codeasashu** [kenjis/ci-phpunit-test#377]
Getting this error as soon as I enabled monkey patching.

```
A PHP Error was encountered

Severity:    Warning
Message:     flock(): Illegal operation argument
Filename:    /Users/ashutoshchaudhary/code/office/php/asterisk/engine-obd/application/tests/_ci_phpunit_test/patcher/IncludeStream.php
Line Number: 212
```

**Comment by codeasashu** [comment]
@kenjis PR #378 seems to fix this. Kindly review

**Comment by kenjis** [comment]
@codeasashu If you know what code causes the error, could you tell me what it is?

**Comment by kenjis** [comment]
Fixed in #378
